include/zenoh/api/id.hxx

```diff
@@ -34,16 +34,21 @@ class Id : public Copyable<::z_id_t> {
    public:
     /// @name Methods
 
-    /// Check if the ID is valid.
-    /// @return ``true`` if the ID is valid.
-    bool is_valid() const { return _0.id[0] != 0; }
-
-    /// Return the byte sequence of the ID.
+    /// Return the byte sequence of the 'Id'.
     const std::array<uint8_t, 16>& bytes() const { return *reinterpret_cast<const std::array<uint8_t, 16>*>(&_0.id); }
+
+    /// @brief Formats the `Id` into 16-digit hex string (LSB-first order).
+    std::string to_string() const {
+        ::z_owned_string_t s;
+        ::z_id_to_string(interop::as_copyable_c_ptr(*this), &s);
+        std::string ss(::z_string_data(::z_loan(s)), ::z_string_len(::z_loan(s)));
+        ::z_drop(::z_move(s));
+        return ss;
+    }
 };
 
 /// @warning This API has been marked as unstable: it works as advertised, but it may be changed in a future release.
-/// @brief Print ``Id`` in the hex format.
+/// @brief Print `Id` in the hex format.
```

Comment:
Why Id is put to single backquotes while other identifiers below a corrected in the opposite way, to double quotes.
Also these is 'Id' in single-quotes (not backquotes) above